### PR TITLE
add code to two_sum function

### PR DESCRIPTION
**Pull request by soridalac**
add function two_sum and test

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```
Comment (amelntamark) [inline functions/two_sum.py:2]: this function looks good! perhaps we could add a docstring to describe its functionality.

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [1, 2]
+
+
+def test_returns_0_1_for_3_3_with_target_6():
```
Comment (amelntamark) [inline tests/two_sum_test.py:28]: these tests look great-they have really clear arrange, act, and assert steps. we could probably add some edge case tests too!

File: tests/two_sum_test.py [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+    # Arrange
+    n = [3, 3]
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```
Comment (amelntamark) [inline tests/two_sum_test.py:37]:
```suggestion
    assert answer == [0, 1]
    
def test_empty_list_returns_empty_list():
    n = []
    t = 2
    
    answer = twoSum(n, t)
    
    assert answer == []
```

**Review by amelntamark — COMMENTED** [review]
Everything's working and the style is super clean! I suggested an additional test for checking edge cases. It might also be nice to add a docstring to the function twoSum to describe its functionality.
